ValveResourceFormat/IO/ModelExtract.cs

```diff
@@ -268,6 +270,42 @@ Lazy<KVObject> MakeLazyList(string className)
 
                 AddItem(renderMeshList.Value, renderMeshFile);
             }
+
+            var mesh = RenderMeshesToExtract.First();
```

Comment:
Any idea if the attachments ever change through meshes on the same model? Found it odd that they hold that data in MDAT when it doesn't change.

Reply:
I don't think they can change, I've always thought that they're just a part of the overall model. It's kind of implied by ModelDoc's UI too.